odfuzz/entities.py

```diff
@@ -109,11 +109,12 @@ def _append_corresponding_queryable(self, query_group):
 
 class DirectBuilder:
     """A class for building and initializing all queryable entities with metadata passed in constructor."""
-    def __init__(self, metadata, restrictions):
+    def __init__(self, metadata, restrictions,method):
         self._queryable = QueryableEntities()
         self._metadata_string = metadata
         self._restrictions = restrictions
 
+        os.environ["ODFUZZ_HTTP_METHOD"]=method
```

Comment:
remove. DirectBuilder is never be called with environment variables configuration. It will be provided only when creating new class instance in the integration kind of usage.

Reply:
ok. Adding a setter for the FuzzerConfig class instead